course02/homework01/joost/linkedlist.cc

```diff
@@ -0,0 +1,48 @@
+#include "linkedlist.h"
+#include <iostream>
+#include <cassert>
+
+LinkedListInt::LinkedListInt():
+	mHead(NULL),
+	mSize(0)
+{}
+
+int LinkedListInt::size() const
+{
+	return mSize;
+}
+
+void LinkedListInt::append(int value)
+{
+	Node* last = mHead;
+	if( last != NULL )
+	{
+		while( last->next != NULL )
+		{
+			last = last->next;
+		}
+		last->next = new Node();
+		last->next->value = value;
```

Comment:
This should typically be done in Node's ctor. On the top of this, if you provide a user defined ctor for Node, the default ctor wouldn't be generated and it thus guarantees that you will never forget to make this assignment...!